data/json/recipes/recipe_weapon.json

```diff
@@ -6733,5 +6733,52 @@
     ["steel_chunk", 4],
     ["scrap", 12]]
   ]
+},
+{
+  "type" : "recipe",
+  "result": "chemical_thrower",
+  "category": "CC_WEAPON",
+  "subcategory": "CSC_WEAPON_RANGED",
+  "skill_used": "mechanics",
+  "skills_required": [[ "gun", 3 ], [ "cooking", 3]],
+  "difficulty": 7,
```

Comment:
Fire does it very cheaply. Digging holes into fungal bed is a work-intensive, but precise method of fixing it.